FlashEnglish/Sources/Views/Home/HomeView.swift

```diff
@@ -8,22 +8,34 @@
 import SwiftUI
 
 struct HomeView: View {
+
+
+
+
+
+
     // MARK: - Properties
     @EnvironmentObject var quizManager: QuizManager
     @EnvironmentObject var navigationManager: NavigationManager
     @Binding var presentSideMenu: Bool
-    private let levelItem: [LevelItem] = [
+    private let levelItem: [LevelItem] =     [
         LevelItem(image: Asset.Assets.imgJuniorHighSchool.swiftUIImage, title: "中学生レベル", levelCase: .juniorHighSchool),
         LevelItem(image: Asset.Assets.imgHighSchool.swiftUIImage, title: "高校生レベル", levelCase: .highSchool),
         LevelItem(image: Asset.Assets.imgCollege.swiftUIImage, title: "大学生レベル", levelCase: .college),
         LevelItem(image: Asset.Assets.imgBusinessman.swiftUIImage, title: "社会人レベル", levelCase: .businessman),
         LevelItem(image: Asset.Assets.imgExpert.swiftUIImage, title: "専門家レベル", levelCase: .expert),
         LevelItem(image: Asset.Assets.imgMonster.swiftUIImage, title: "人外レベル", levelCase: .monster)
     ]
-    private let gridItems = [GridItem(.flexible()), GridItem(.flexible())]
+          private let gridItems = [GridItem(.flexible()), GridItem(.flexible())]
 
     // MARK: - Body
     var body: some View {
+
+
+
+
+
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/0e74ba265c47132c04f133da09675f0b9e81baab/FlashEnglish/Sources/Views/Home/HomeView.swift#L38"></span>Limit vertical whitespace to a single empty line; currently 6<br />
<code>vertical_whitespace</code> <code>HomeView.swift:38</code></td>
    </tr>
  </tbody>
</table>